### PR TITLE
Rename raw to raws; closes #54

### LINKED ISSUE #54: raws versus raw
Wanted to raise this slight discrepancy with the PostCSS main parser. postcss-selector-parser uses a property name of `raw` on its nodes, while PostCSS uses the property name `raws` on its nodes. Surely nothing critical, but would be keen to see a consensus reached so this could be merged as part of PostCSS proper, and also for ease of development in other tools.

### PR DESCRIPTION
**Pull request by davidtheclark**
After this can 2.0 be released? Or is there something else blocking?

### CODE REVIEW

**Comment by ben-eb** [comment]
Can we update https://github.com/postcss/postcss-selector-parser/blob/master/API.md too? There isn't a documented section for this property, that should be rectified. 😃

**Comment by davidtheclark** [comment]
When I added the quotation mark business to attributes I added this: https://github.com/postcss/postcss-selector-parser/blob/master/API.md#attribute-nodes. Do you want a generic section for all nodes? (Are there other raw properties? From searching I only found them on attribute nodes I *think*.)

**Comment by davidtheclark** [comment]
I see there is also an `attribute.raws.insensitive` property but can't tell at a glance what it's for. Could you summarize and I'll write that up?

Anything else you want written up for `raws`?

**Comment by ben-eb** [comment]
We could do all nodes too, sure. 👍

`attribute.raws.insensitive` contains extraneous whitespace around the `i`. I can't remember if this is legal or not in CSS as it's been a while since writing this code.

https://github.com/postcss/postcss-selector-parser/blob/master/src/__tests__/attributes.js#L228-L231

**Comment by davidtheclark** [comment]
Ok, added a section for `attribute.raws.insensitive`.
